project/core/urls.py

```diff
@@ -20,10 +20,8 @@
 from django.conf import settings
 from django.views.static import serve
 from django.views.generic.base import RedirectView
-
 from core.router import CiviWikiRouter
-
-from api import urls as api
+from threads import urls as api
```

Comment:
This should be `threads_urls` now instead of `api`
```suggestion
from threads import urls as threads_urls
```